src/pages/login.tsx

```diff
@@ -31,7 +31,7 @@ export default function Login() {
         });
         return;
       }
-      void router.push("/dashboard");
+      void router.push("/live");
```

Comment:
can we keep this as redirect to dashboard until live site is ready

Reply:
fixed